xline/src/storage/kv_store.rs

```diff
@@ -408,56 +474,17 @@ where
         limit: usize,
         count_only: bool,
     ) -> Result<(Vec<KeyValue>, usize), ExecuteError> {
-        let mut revisions = self.index.get(key, range_end, revision);
```

Comment:
Could this method be moved into `KvStoreInner`?

Reply:
Moved